lib/ExtensionPoint/Readme.md

```diff
@@ -11,7 +11,7 @@ Render default content when no extension found
 
 Render an extension
 
-<small>Try to change the `extensionName` to `Hello1` to see the default rendering provided by the Extension Point.</small>
+<small>Try to change the `extensionName` to `Hello` to see the default rendering provided by the Extension Point.</small>
```

Comment:
Changing it to `Hello` will not show the default rendering, since the Extension is named "Hello". Try it out :)

Reply:
I got it now. I though it was describing the code below.  
I forgot that we can directly play with the code from the doc page, now it makes sense. :)